src/RpcClient/ContractClient.cs

```diff
@@ -30,10 +32,10 @@ public ContractClient(RpcClient rpc)
         /// <param name="operation">contract operation</param>
         /// <param name="args">operation arguments</param>
         /// <returns></returns>
-        public RpcInvokeResult TestInvoke(UInt160 scriptHash, string operation, params object[] args)
+        public Task<RpcInvokeResult> TestInvokeAsync(UInt160 scriptHash, string operation, params object[] args)
```

Comment:
This one is not `public async`?

Reply:
no, because it doesn't use await. Since TestInvoke doesn't need to process the results of InvokeScriptAsync, it can simply return the Task<RpcInvokeResult> returned by InvokeScriptAsync